src/routes/__layout.svelte

```diff
@@ -35,5 +35,5 @@
 		display: flex;
 		flex-direction: row;
 		justify-content: end;
-	}
+	} */
```

Comment:
Please delete commented selectors, as we are using GIt, there is no point in keeping dead code.